src/services/item/plugins/itemVisibility/index.ts

```diff
@@ -12,54 +12,56 @@ import { validatedMemberAccountRole } from '../../../member/strategies/validated
 import { Item } from '../../entities/Item';
 import { ItemService } from '../../service';
 import { create, deleteOne } from './schemas';
-import { ItemTagService } from './service';
+import { ItemVisibilityService } from './service';
 
 /**
- * Item tag plugin
- * Dynamic tag behavior for simple cases
+ * Item Visibility plugin
+ * Dynamic visibility behavior for simple cases
  * - public:
  * - hidden
  *
  * The feature should be stackable and be inherited (a parent public and an item public is allowed)
- * The tag can be copied alongside the item
+ * The visibility can be copied alongside the item
```

Comment:
Since there is a hook, I think anything that uses the copy will be copied. I don't remember the state currently, but I won't change the behavior in this PR.